src/main/java/jek/collections/IntArrayList.java

```diff
@@ -0,0 +1,155 @@
+package jek.collections;
+
+import java.util.Collection;
+import java.util.Iterator;
+import java.util.List;
+import java.util.ListIterator;
+
+public class IntArrayList implements List {
+
+    private int end;
+    private Object[] ial;
+
+    public IntArrayList() {
+        ial = new Object[10];
+        end = 0;
+    }
+
+    @Override
+    public int size() {
+        return end;
+    }
+
+    @Override
+    public boolean isEmpty() {
+        if (end == 0)
```

Comment:
the stuff you put in braces is actually the boolean value and can be used as return values. So for your case the method can be simplified to 
return (end == 0);